sqlmesh/core/node.py

```diff
@@ -193,6 +200,8 @@ class _Node(PydanticModel):
     tags: t.List[str] = []
     stamp: t.Optional[str] = None
     _path: Path = Path()
+    _data_hash: t.Optional[str] = None
```

Comment:
Why is this here and not in the `_Model`? I think it only applies there.

Reply:
no, audits also have these properties